### DISCUSSION TITLE
Unimplemented Instruction (LLIL)

### DISCUSSION THREAD

**Discussion by hsreina** [Vector35/binaryninja-api#4976] [Analysis] [answered]
Hi,

Is there anything I can do for unimplemented instructions? I am working on a project based on arm7.
<img width="471" alt="image" src="https://github.com/Vector35/binaryninja-api/assets/7342613/da3041f3-495f-4e93-a5dc-b01b999ed02e">
Are they already implemented in some way and I just didn't choose the correct options while opening the binary 
or are they going to be implemented at some point?
or can I maybe do something to implement them?

Thank you.

**Comment by plafosse** [comment, marked as answer]
Feel free to report instructions that are marked "Unimplemented" and are causing you analysis issues in our issue tracker. Short of that our architecture plugins are open source and you do have the option of fixing them yourself. https://github.com/Vector35/arch-armv7  We do accept pull requests :)
